CMakeLists.txt

```diff
@@ -112,6 +113,17 @@ if (MATERIALX_BUILD_JS)
     set(MATERIALX_BUILD_TESTS OFF)
 endif()
 
+# Validate MSL-only rendering option
+if(MATERIALX_RENDER_MSL_ONLY)
```

Comment:
Turn off GLSL rendering. For some reason there a renderGLSL module dependency for renderMSL. Not touching that dependency here.